### PR TITLE
fix(tests): load only required flows for runner tests

### PR DESCRIPTION
**Pull request by nkwiatkowski**
### What changes are being made and why?

---

### How the changes have been QAed?

---

### CODE REVIEW

File: jdbc/src/test/java/io/kestra/jdbc/loader/FlowLoader.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+            executor.execute();
+        } catch (Exception e){
+            throw e;
+        } finally {
+            flowRepository.findAllForAllTenants().forEach(flow -> flowRepository.delete(
+                FlowWithSource.of(flow, "unused")));
+        }
+    }
+
+    public interface Executor {
```
Comment (loicmathieu) [inline jdbc/src/test/java/io/kestra/jdbc/loader/FlowLoader.java]: Maybe add `@FunctionalInterface` ? 
Or better use Rethrow.RunnableChecked it should cover what you need here

**Review by loicmathieu — APPROVED** [review]
LGTM, neat!

File: jdbc/src/test/java/io/kestra/jdbc/loader/FlowLoader.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+import io.kestra.core.repositories.LocalFlowRepositoryLoader;
+import io.kestra.core.utils.Rethrow.RunnableChecked;
+import io.kestra.core.utils.TestsUtils;
+import jakarta.inject.Inject;
+import jakarta.inject.Singleton;
+import java.util.List;
+import java.util.Objects;
+
+@Singleton
+public class FlowLoader {
```
Comment (tchiotludo) [inline jdbc/src/test/java/io/kestra/jdbc/loader/FlowLoader.java]: maybe on the TestUtils directly no ?

File: core/src/test/resources/flows/valids/trigger-flow-listener-namespace-condition-fail.yaml [hunk 3]
```diff
@@ -0,0 +1,21 @@
+id: trigger-flow-listener-namespace-condition-fail
+namespace: io.kestra.tests.trigger.fail
```
Comment (tchiotludo) [inline core/src/test/resources/flows/valids/trigger-flow-listener-namespace-condition-fail.yaml]: Do we still need to change the namespace here ?

Comment (nkwiatkowski) [reply core/src/test/resources/flows/valids/trigger-flow-listener-namespace-condition-fail.yaml]: I was surprised, but yes. I still had false trigger in this specific test

**Pull request merged**